docs/backup_restore/catalog-db-restore.sh

```diff
@@ -23,16 +24,12 @@ cf set-env backup-manager DATASTORE_S3_SERVICE_NAME backup-manager-s3
 # Go to the correct space
 cf target -s $space_name
 
-# # create temp Database
-if [[ "${space_name}" == 'prod' ]]; then
-  db_plan=large-gp-psql-redundant
-else
-  db_plan=large-gp-psql
-fi
-cf create-service aws-rds ${db_plan} catalog-db-new -c "{\"storage\": ${storage_size}, \"version\": \"12\"}" --wait
+cf create-service aws-rds ${db_plan} catalog-db-new -c "{\"storage\": ${storage_size}, \"version\": \"15\"}" --wait
 cf bind-service backup-manager catalog-db-new
 cf restart backup-manager
 
+cf scale backup-manager -i 1
```

Comment:
was getting an ssh error with the app when 0 instances are up so i scale up and down